docs/sources/articles/host_integration.md

```diff
@@ -65,11 +65,14 @@ a new service that will be started after the docker daemon service has started.
     [Unit]
     Description=Redis container
     Author=Me
+    Requires=docker.service
     After=docker.service
 
     [Service]
     Restart=always
     ExecStart=/usr/bin/docker start -a redis_server
+    # for more options, use 'run' instead of 'start', but not suggested
```

Comment:
actually, no, this is a code-block, so not backticks

I think the `# for more options ...` comment could be rephrased as `# if you need to pass options to the redis container (such as '--env'), then you'll need to use 'docker run' rather than 'docker start'`